codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -374,7 +369,7 @@ impl EventMod {
             None => "".to_string(),
             Some(fuel_event_kind_code) => format!(
                 r#"
-let config: fuelEventConfig = {{
+let register = (): fuelEventConfig => {{
```

Comment:
I changed it to function, so the handlerRegister is up to date

Reply:
Otherwise it breaks wildcard mode for Fuel

Reply:
Nice, yeah that's why the EVM side handles all this in the RegisterHandlers file after handlers are registered.